server/devpi_server/__version__.py

```diff
@@ -0,0 +1,8 @@
+
+__title__ = 'devpi-server'
+__description__ = 'devpi-server: reliable private and pypi.org caching server'
+__url__ = 'http://doc.devpi.net'
+__version__ = '4.8.1.dev0'
+__maintainer__ = 'Kenneth Reitz'
```

Comment:
wtf

Reply:
for sake of the time, I copied requests module and just change the values